docs/install/docker-install/quick-start.md

```
@@ -4,89 +4,142 @@ tags:
 
-2. Grab a pair of client_id and client_pw(secret) from `setup.properties` or `/opt/jans/jans-setup/setup.properties.last`
+2. To connect to the TUI, you need your FQDN, Client ID, and Client Secret. Since the AIO deployment stores configurations in Consul and Vault, you can extract these credentials directly from the running Docker containers. 
 
-3. Use the CLI tools located under `/opt/jans/jans-cli/` to configure Gluu flex as needed. For example you can run the [TUI](https://docs.jans.io/head/admin/config-guide/config-tools/jans-tui/):
-    ```bash
-    python3 /opt/jans/jans-cli/config-cli-tui.py
+    Run the following commands on your host machine to store the credentials in environment variables:
+
+    ```bash title="Replace with your actual FQDN"
+    FQDN="demoexample.gluu.org"
+    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)